CDCgov/cdc-open-viz
# Feature/dev 8160

## Testing Steps
1. Open the editor Panel > Filters Accordion
2. Check the box to keep filter styles on mobile
3. Ensure desktop styles are applied on mobile screens




## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
